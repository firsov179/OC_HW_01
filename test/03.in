1, 2 - Buckle my shoe,
3, 4 - Knock at the door,
5, 6 - Pick up sticks,
7, 8 - Lay them straight,
9, 10 - A big fat hen,
11, 12 - Dig and delve,
13, 14 - Maids a-courting,
15, 16 - Maids in the kitchen,
17, 18 - Maids a-waiting,
19, 20 - Iâ€™ve had plenty!