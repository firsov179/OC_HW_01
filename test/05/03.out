1: 12
,: 19
 : 52
2: 3
-: 13
B: 1
u: 3
c: 6
k: 5
l: 3
e: 11
m: 2
y: 3
s: 7
h: 8
o: 5
3: 2
4: 2
K: 1
n: 8
a: 12
t: 12
d: 7
r: 3
5: 2
6: 2
P: 1
i: 13
p: 2
7: 2
8: 2
L: 1
g: 5
9: 2
0: 2
A: 1
b: 1
f: 1
D: 1
v: 2
M: 3
w: 1
I: 2
!: 1
